### PR TITLE
Split clear tests out by environment to fix tests

### PR DESCRIPTION
**Pull request by SonarBeserk**
This in theory signals to the clear command that the typical x11 terminal is set. The CI runner does not appear to be setting this value but it should still be installed.

### CODE REVIEW

**Comment by dixonwille** [comment]
Can you add Windows to the matrix to run against? I updated linux and darwin to not use exec.Command

File: .github/workflows/pullRequests.yaml [hunk 1]
```diff
@@ -20,6 +20,7 @@ jobs:
     strategy:
       matrix:
         dir: [".", "example"]
+        os: [ubuntu-latest, windows-latest, macos-latest]
     runs-on: ubuntu-latest
```
Comment (dixonwille) [inline .github/workflows/pullRequests.yaml:24]: doesn't this need to run with the matrix variable

Comment (SonarBeserk) [reply .github/workflows/pullRequests.yaml]: Good catch, fixed

**Review by dixonwille — APPROVED** [review]

**Comment by SonarBeserk** [comment]
This is definitely one of those technically works solutions. The actions runs besides windows are quick at least.

**Pull request merged**